pkgs/servers/keycloak/scim-keycloak-user-storage-spi/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, maven
+, javaPackages
+}:
+
+javaPackages.mavenfod rec {
+  pname = "scim-keycloak-user-storage-spi";
+  version = "beta";
+
+  src = fetchFromGitHub {
+    owner = "justin-stephenson";
+    repo = "scim-keycloak-user-storage-spi";
+    rev = "1be97049edf096ca0d9a78d988623d5d3f310fb1";
+    hash = "sha256-aGHInyy+VgyfjrXeZ6T6jfI00xaCwrRTehnew+mWYnQ=";
+  };
+
+  mvnHash = "sha256-CK42d+Ta1/XNQWCLaje6sI+C90YvzUcteuasVkUPfCk=";
+
+  nativeBuildInputs = [
+    maven
+  ];
+
+  installPhase = ''
+    JAR=$(find target -iname "*.jar")
+    install -D "$JAR" "$out/$(basename $JAR)"
```

Comment:
The jar file probably has a stable location. Please use that instead.

Reply:
done.